modules/mod_breadcrumbs/tmpl/default.php

```diff
@@ -78,14 +78,27 @@
 
 	foreach ($list as $key => $item)
 	{
-		$data['itemListElement'][] = [
-				'@type'    => 'ListItem',
-				'position' => $key + 1,
-				'item'     => [
-						'@id'  => $item->link ? Route::_($item->link, true, Route::TLS_IGNORE, true) : Route::_(Uri::getInstance()),
-						'name' => $item->name
-				]
-		];
+		// Only add item to JSON if it has a valid link, otherwise skip it.
+		if(!empty($item->link)) {
+			$data['itemListElement'][] = [
+					'@type'    => 'ListItem',
+					'position' => $key + 1,
+					'item'     => [
+							'@id'  => $item->link ? Route::_($item->link, true, Route::TLS_IGNORE, true) : Route::_(Uri::getInstance()),
+							'name' => $item->name
+					]
```

Comment:
```suggestion
							'name' => $item->name,
					],
```
The code style checker skips layout files, but in the pure PHP parts we still should respect our code style.